.travis.yml

```diff
@@ -12,9 +12,23 @@ matrix:
     addons:
       apt:
         packages:
+          - pkg-config
           - libevent-dev
+
+install:
+  - pyenv global system 3.6
+  - pip3 install 'meson>=0.45'
```

Comment:
Based on previous experiences with meson and its instability, I recommend locking the version.